Platform/AlderlakeBoardPkg/Library/Stage2BoardInitLib/CpuInfoLib.c

```diff
@@ -95,7 +96,7 @@ GetCpuSkuInfo (
   }
 
   if (!SkuFound) {
-    DEBUG ((DEBUG_ERROR, "Unsupported CPU SKU, Device ID: 0x%02X, CPUID: 0x%08X!\n", CpuDid, CpuFamilyModel));
+    DEBUG ((DEBUG_INFO, "Unsupported CPU SKU, Device ID: 0x%02X, CPUID: 0x%08X!\n", CpuDid, CpuFamilyModel));
```

Comment:
don't we want this to be "DEBUG_ERROR" instead ?

Reply:
Yes, Better to keep this as ERROR level. will update it.